R/PlainTextParam.R

```diff
@@ -9,76 +10,79 @@
 #' @family xcms result export and import formats.
 #'
 #' @description
-#' The `PlainTextParam` class and method enable users to save/load an
-#' `MsExperiment` or `XcmsExperiment` object as a collections of plain text
-#' files in/from a specified folder.
-#' Note that, while for all xcms results within the `XcmsExperiment` can
-#' and will be exported, the full raw MS data (of the object's `Spectra` object)
-#' will currently not be exported in plain text format. For `Spectra` using the
-#' [MsBackendMzR()] backend, the names of the raw data files will however be
-#' exported (which enables to *load* the full `Spectra` respectively
-#' `MsExperiment` objects).
-#'
-#' For an `MsExperiment` object, the exported files include:
 #'
-#' - The [sampleData()] stored as a text file named *sample_data.txt*.
+#' The `storeResults()` and `loadResults()` methods with the `PlainTextParam`
```

Comment:
I restructured the documentation a bit, focusing more on the `MsExperiment` and `XcmsExperiment` and moving the ones on `Spectra` and `MsBackendMzR` a bit down - also to make it easier to eventually later move these methods to the *Spectra* package.